src/tests/units/operator_tests.cpp

```diff
@@ -297,13 +328,14 @@ void tests(DeviceType device, Type floatType = Type::float32) {
     s2->val()->get(values); CHECK(values == vS2);
 
     CHECK(m3->val()->scalar() == 9);
+    auto floatApproxLocal = [](T x, T y) -> bool { return x == Approx(y).margin(0.004); };
```

Comment:
Just curious. Why 0.004?

Reply:
Ah - this test was failing on my Titan V so I just increased the margin so that it passed. Should I revert this?

Reply:
Got it. No need to revert. Thanks!

Reply:
Could you please add a short comment to the code with that explanation?

Reply:
Done